actions/common.go

```diff
@@ -26,6 +26,7 @@ var (
 		"router":           {"Router"},
 		"slugbuilder":      {"SlugBuilder"},
 		"slugrunner":       {"SlugRunner"},
+    "stdout-metrics":   {"StdoutMetrics"}
```

Comment:
![screen shot 2016-05-09 at 2 03 47 pm](https://cloud.githubusercontent.com/assets/430360/15126359/e1fb1dbe-15ee-11e6-9abb-8ea9d16dca36.png)


not sure wtf is up